app/mailers/participant_mailer.rb

```diff
@@ -0,0 +1,44 @@
+class ParticipantMailer < ApplicationMailer
+
+    def send_reminder(interview_id,time)
+        # puts "time from mailer"
+        # puts time
+        @interview = Interview.find(interview_id)
+        # puts @interview.updated_at
+        # puts "updated time"
+        if(@interview.updated_at <= time)
+            # puts "Yass"
+            @interviewees = Interviewee.where("id in (?)", @interview.interviewee_ids)
+            emails = []
+            for i in @interviewees do 
```

Comment:
for interviewee in @interviewees do , always try to use meaningful variables.

Reply:
commit - 64a01bd